add more installation details (#5)
swinton/github-app-repl: README.md

npm install -D github-app-repl
```

> It expects a `APP_ID` environment variable, and corresponding `.pem` file. Most Probot projects should have this. 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> To find these details for a GitHub App you have installed, log into GitHub and from your profile view go to `Settings > Developer Settings > GitHub Apps > YOUR_APP > EDIT`

Reviewer: ```suggestion
```

> From here, you can generate a new private key if you'd like. You will need either need to move the `.pem` file into `gitHub-app-repl`'s directory or update the `.env` file to include a file path to your key. 

Reviewer: ```suggestion
## Configuration

A couple of environment variables are required for configuration:

1. **`APP_ID`**: This should be set to the ID of your GitHub App. 
    - This is available from the settings page of your GitHub App, as `App ID`.
1. **`PRIVATE_KEY_PATH`**: This should be set to a  local filesystem path to a private key for your GitHub App. 
    - More information on generating and downloading private keys for your GitHub Apps is [available in the documentation](https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#generating-a-private-key).

These environment variables can either be set directly, or via a `.env` file that follows [this example](https://github.com/swinton/github-app-repl/blob/master/.env.example).
```
